tests/test_rootfs.py

```diff
@@ -54,7 +54,7 @@ def verify_fstab(data):
             occurred[cols[1]] = True
 
     @pytest.mark.only_with_image("ext4", "ext3", "ext2")
-    @pytest.mark.min_mender_client_version("1.0.0")
+    @pytest.mark.min_mender_version("1.0.0")
     def test_expected_files_ext234(
```

Comment:
As we are reusing an existing test, we need to bump `min_mender_version` to 2.5.0

Reply:
Alternatively we could have a separate test, but I have no inconvenient in reusing and skipping this test for upcoming bugfix releases, as this is very low risk.

Reply:
Good catch, will fix!